Rain_Water_Trapping/Rain_Water_Trapping.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h> 
+
+using namespace std;
+
+//function to calculate min
+int min(int a, int b)
+{
+    return a < b ? a :b;
+}
+
+
+//function to calculate max
+int max(int a, int b)
+{
+    return a > b ? a :b;
+}
+
+ 

```

Comment:
Remove extra lines between functions. Only one blank line should be present between two functions.